tests/rptest/tests/librdkafka_test.py

```diff
@@ -11,14 +11,52 @@
 import os
 
 from rptest.services.cluster import cluster
-from ducktape.mark import matrix, ignore
+from ducktape.mark import matrix
 
 from rptest.services.librdkafka_test_case import LibrdkafkaTestcase
 from rptest.tests.redpanda_test import RedpandaTest
 
 from rptest.utils.mode_checks import skip_debug_mode
 
 
+def tests_to_run():
+    ignored_tests = set([
+        14,
+        51,
+        67,
```

Comment:
I looked at the failures from last night:
```
grep FAIL.*rdkaf /tmp/8h.log 
FAIL test: LibrdkafkaTest.test_librdkafka.test_num=26 (1/12 runs)
FAIL test: LibrdkafkaTest.test_librdkafka.test_num=86 (1/12 runs)
FAIL test: LibrdkafkaTest.test_librdkafka.test_num=91 (1/12 runs)
FAIL test: LibrdkafkaTest.test_librdkafka.test_num=93 (1/12 runs)
```

Can we add those to this list with notes to investigate why they are failing later?

Reply:
I will mark the test ok_to_fail in follow up PR.